### PR TITLE
ENH: Add the logic to use galvo_shutter

### PR DESCRIPTION
**Pull request by mrakitin**
WIP: Under testing at the beamline.
- [x] Fix the issue with opened shutter while moving.
- [x] Pass the shutter in `def align_ht(self)`.
- [x] Integrate the galvo-shutter into the scanning logic.

### CODE REVIEW

File: startup/11-shutters.py [hunk 1]
```diff
@@ -52,6 +52,8 @@ def shutter_cb(value, timestamp, **kwargs):
         return st
 
     def stop(self, success):
+        # MR: commenting out on 10/30/2018 for manual control of the shutters (see #10).
+        '''
```
Comment (tacaswell) [inline startup/11-shutters.py]: Can you put in leading `#` instead of string comments?  This makes the diffs very hard to read....

**Comment by tacaswell** [comment]
The reason that `stop` exists in that way is so that the shutters would close when the plan is suspended (either via suspenders or ctrl-C).  I think we can get that functionality via the `pause` and `resume` methods.

File: startup/11-shutters.py (hunk 1 above)
Comment (mrakitin) [reply startup/11-shutters.py]: Sure, I will. Thanks @tacaswell!

**Comment by mrakitin** [comment]
@tacaswell, 094d566 takes care of the proper commenting.

> we can get that functionality via the `pause` and `resume` methods.

Can you clarify? Should that commented code go to the `pause` and `resume` methods? We already have the `resume` method. What if I just rename the `stop` method to the `pause` one?

**Comment by tacaswell** [comment]
> What if I just rename the `stop` method to the `pause` one?

I think that will work.

**Comment by mrakitin** [comment]
The hardware was removed from the beamline, so it's not needed for now. We'll keep the PR branch in case it ever comes back.

**Pull request closed without merging**